### DISCUSSION TITLE
weird characters after build

### DISCUSSION THREAD

**Discussion by Belouccio** [parcel-bundler/parcel#6553] [Q&A]
Hi there! I use parcel to build my preact app. After build, some symbols (cyrrilic) randomly replaces to ��, you can see that on the screenshot below. How can i fix that, any ideas?

parcel verrsion: 'next'
preact version: 10.5.14
system: ubuntu 16

![image](https://user-images.githubusercontent.com/25301055/124642520-9a4c0280-de98-11eb-916d-adb663d9c1de.png)

**Comment by mischnic** [comment]
Can you share the original word (before building)?

**Reply by Belouccio** [reply]
Сотрудников

**Reply by Belouccio** [reply]
it randomply replace symbols after each build, if i run build command again (without any changes), this symbol will be nice, but other will broken

**Reply by Belouccio** [reply]
another example
конфиденциальности -> ко��фиденциальности

but in sibling word this symbol transpile correctly (Лицензионный) symbol at position 4

**Reply by Belouccio** [reply]
what interesting, if i use google translate and translate this page, this symbols will be nice, maybe this help

![image](https://user-images.githubusercontent.com/25301055/124645304-f2d0cf00-de9b-11eb-89cb-c251bbf570cc.png)

**Reply by Belouccio** [reply]
any news?
